src/dmd/probing.d

```diff
@@ -0,0 +1,207 @@
+/**
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * This modules helps to create a list of supported features (aka probing output).
+ * The probing information is returned in JSON.
+ *
+ * Copyright:   Copyright (c) 1999-2017 by The D Language Foundation, All Rights Reserved
+ * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
+ * License:     $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/mars.d, _mars.d)
+ * Documentation:  https://dlang.org/phobos/dmd_mars.html
+ * Coverage:    https://codecov.io/gh/dlang/dmd/src/master/src/dmd/mars.d
```

Comment:
This documentation header seems to be cut and pasted from `mars.d`, but needs to be updated for this file.